src/Picqer/Financials/Exact/BankEntry.php

```diff
@@ -58,7 +58,7 @@ class BankEntry extends Model
 
     public function addItem(array $array)
     {
-        if (! isset($this->attributes['BankEntryLines']) || $this->attributes['BankEntryLines'] == null) {
+        if ( ! isset($this->attributes['BankEntryLines']) || $this->attributes['BankEntryLines'] == null) {
```

Comment:
Why this should have the additional white space before the `!ieest`?

Is there any coding style rule on this repository?